lib/services/messenger/translators/flowdock.ts

```diff
@@ -521,6 +500,71 @@ export class FlowdockTranslator extends TranslatorScaffold implements Translator
 		}
 		return connectionDetails;
 	}
+
+	/**
+	 * Promise to retrieve the alias used to present an event
+	 * @param event  Event to scrutinise for user details
+	 * @returns      Promise that resolves to the alias presented with the message
+	 */
+	protected fetchAlias(event: FlowdockEvent): Promise<string> {
+		if (event.rawEvent.external_user_name) {
+			return event.rawEvent.external_user_name;
+		}
+		const url = `/organizations/${this.organization}/users/${event.rawEvent.user}`;
+		return FlowdockTranslator.fetchFromSession(this.session, url)
+		.then((user) => {
```

Comment:
`.get('nick');` could be used instead.

Reply:
Wow, that is an incredible function, and one I will use again!